src/helpers/asyncPipe.ts

```diff
@@ -1,10 +1,13 @@
+import { ux } from '@cto.ai/sdk'
+
 // based on https://github.com/obengwilliam/pipeawait
 
 const asyncPipe = (...fns: Function[]) => (param?: any) =>
   fns.reduce(async (acc, fn) => fn(await acc), param)
 
 const _trace = (msg: string) => (x: any) => {
-  console.log(msg, x)
+  //TODO: this should/should not be async
```

Comment:
Reverted back to ux.print, since:

- _trace is used only for debugging
- _trace is not an async function, and it would have to be converted to async for awaiting ux.print